app/js/components/manageApps/ManageApps.jsx

```diff
@@ -813,9 +838,9 @@ export default class ManageApps extends React.Component {
                   <span className="glyphicon glyphicon-play" />
                   Start All Modules
                 </span>
-                <Link to="/manageSettings" className="manage-settings-button">
-                  <i className="glyphicon glyphicon-cog settings-icon" id="setting-icon-btn" />
-                </Link>
+                <span style={{position: 'relative'}} ref="target"
```

Comment:
In order to keep the styling consistent, consider making a class for this instead since all the components are using external stylesheets and not inline styles.